packages/test/src/util/browser-tests-runner/bundler.js

```diff
@@ -0,0 +1,136 @@
+const fs = require("mz/fs");
+const path = require("path");
+const lasso = require("lasso");
+const resolveFrom = require("resolve-from");
+const Transform = require("stream").Transform;
+const parseRequire = require("lasso-require/src/util/parseRequire");
+const shouldCover = !!process.env.NYC_CONFIG;
+const baseDependencies = [
+  "mocha/mocha.js",
+  "mocha/mocha.css",
+  "require-run: " + require.resolve("./browser-dependencies")
+];
+
+exports.create = async (tests, options) => {
+  const workDir = options.workDir;
+  const outputDir = path.resolve(workDir, "browser-build");
+  const testDependencies = tests.map(toVirtualModule);
+  const additionalDependencies = resolveTestDependencies(
+    options.browserDependencies || [],
+    options.dir
+  );
+  const customLassoOptions = options.lassoOptions;
```

Comment:
Can `options.lassoOptions` be undefined/null? If so, the below code will fail referencing properties on `customLassoOptions`.

Reply:
Good catch, you the real MVP.